R/get-results.r

```diff
@@ -47,7 +47,8 @@ id_scenarios <- function(directory){
         x <- unlist(strsplit(all.dirs[i], split="/"))
         return(x[length(x)])
     })
-    scenarios <- temp.dirs[grepl("[A-Z0-9-]+-[a-z]+",temp.dirs)]
+    scenarios <- temp.dirs[grepl("[A-Z0-9-]{2}-[a-z]+",temp.dirs)]
```

Comment:
That looks great Sean. Thanks for working on this, I know you are really 
busy, but this is the exact reason I put it in a branch and did not just 
change the master even though its a fairly simple change!

On 1/12/2015 04:57 PM, Sean Anderson wrote:
> In R/get-results.r
> <https://github.com/ss3sim/ss3sim/pull/189#discussion-diff-22836450>:
>
>> @@ -47,7 +47,8 @@ id_scenarios <- function(directory){
>>          x <- unlist(strsplit(all.dirs[i], split="/"))
>>          return(x[length(x)])
>>      })
>> -    scenarios <- temp.dirs[grepl("[A-Z0-9-]+-[a-z]+",temp.dirs)]
>> +    scenarios <- temp.dirs[grepl("[A-Z0-9-]{2}-[a-z]+",temp.dirs)]
>
> How's this one?
>
> |grepl("^([A-Z]{1}[0-9]{1}-)+[a-z]+$", "D0-M9-cod")
> [1] TRUE
> grepl("^([A-Z]{1}[0-9]{1}-)+[a-z]+$", "D0-cod")
> [1] TRUE
>> grepl("^([A-Z]{1}[0-9]{1}-)+[a-z]+$", "SS3-cod")
> [1] FALSE
> |
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ss3sim/ss3sim/pull/189/files#r22836450>.
>